.github/workflows/test-dbt-installation-main.yml

```
@@ -0,0 +1,73 @@
+# **what?**
+
+# **why?**
+
+# **when?**
+# This workflow will run on a schedule every night and also can be
+# manually invoked.
+
+name: Dbt Installation Tests
+
+on:
+  workflow_dispatch:
+
+  # run this once per night to ensure no regressions
+  # schedule:
+  #   - cron: "0 9,13,18 * * *" # 9:00, 13:00, 18:00 UTC
+
+jobs:
+  dbt-installation-homebrew:
+    if: ${{ always() }}
+    strategy:
+      fail-fast: false
+      matrix:
+        package:
+          ["dbt-postgres", "dbt-redshift", "dbt-snowflake", "dbt-bigquery"]
+
+    uses: .github/workflows/test-dbt-installation-homebrew.yml
+    with:
+      package_name: ${{ matrix.package }}
+    secrets:
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_TEST_CHANNEL }}
+
+  dbt-installation-pip:
+    if: ${{ always() }}
+    strategy:
+      fail-fast: false
+      matrix:
+        package:
+          ["dbt-postgres", "dbt-redshift", "dbt-snowflake", "dbt-bigquery"]
```

we don't release spark with homebrew but it needs to be included for every other test type
```suggestion
          ["dbt-postgres", "dbt-redshift", "dbt-snowflake", "dbt-bigquery", "dbt-spark"]
```

---

Fixed in 6b144791535b8a1b4ffcb55c97c9a499a03753c4